apis/v1alpha2/httproute_types.go

```diff
@@ -83,9 +83,10 @@ type HTTPRouteSpec struct {
 	// HTTPRoute specified `test.example.com` and `test.example.net`,
 	// `test.example.net` must not be considered for a match.
 	//
-	// If hostnames do not match with the criteria above, then the HTTPRoute is
-	// not accepted, and the implementation must raise an 'Accepted' Condition
-	// with a status of `False` for the target Listener(s).
+	// If hostnames are specified, and none match with the criteria above, then
+	// the HTTPRoute is not accepted, and the implementation must raise an
+	// 'Accepted' Condition with a status of `False` in the corresponding
+	// RouteParentStatus.
```

Comment:
What if the route specifies two hostnames - one satisfied the above criteria but the other one doesn't?

Reply:
> must be at least one intersecting hostname for the HTTPRoute to be

Maybe this line clarifies that?

Reply:
Yep, that's what I was going for. I also hope that "and none match the criteria" helps convey that only one would need to match.